docs/queues.txt

```diff
@@ -91,16 +101,13 @@ how to handle failed jobs:
        a ``mongodb`` connection. The driver uses the default connection
        if a connection is not specified.
 
-   * - ``collection``
+   * - ``table``
      - Name of the MongoDB collection to store failed
        jobs. The value is ``failed_jobs`` by default.
 
-Then, add the service provider in your application's
-``config/app.php`` file:
-
-.. code-block:: php
-
-   MongoDB\Laravel\MongoDBQueueServiceProvider::class,
+The {+odm-short+} automatically provides the
+``MongoDB\Laravel\MongoDBQueueServiceProvider::class`` class as the
```

Comment:
@GromNaN  Isn't `MongoDBQueueServiceProvider` completely removed (as in it doesn't exist anymore) ?

Reply:
Please disregard this comment, I didn't see the base branch